programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -2443,7 +2438,7 @@ static CommandLine get_status_cmd = make_command("tunnel_status", "Get Tunnel St
 static CommandLine delete_id_cmd = make_command("delete", "delete the identities information", "[-i <identity>]",
                                                  "\t-i|--identity\tidentity info that needs to be deleted\n", delete_identity_opts, send_message_to_tunnel_fn);
 static CommandLine add_id_cmd = make_command("add", "enroll and load the identities information", "[-i <identity>]",
-                                                "\t-i|--identity\tidentity info that needs to be enabled\n"
+                                                "\t-i|--identity\tidentity filename or jwt filename (without path)\n"
```

Comment:
What does this actually mean? "filename or jwt filename"? from looking at the code I would think it must be the jwt filename - with or without the extension (but with the path specified).

Whatever this doc string ends up being, should the one in `delete_id_cmd` follow suit?